<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>actionAdd_prisoner</class>
 <widget class="QWidget" name="actionAdd_prisoner">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>942</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Prisoner</string>
  </property>
  <widget class="QLabel" name="name_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="eye_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Eye Color: </string>
   </property>
  </widget>
  <widget class="QLabel" name="lastName_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Last Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="birthDate_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Birth date: </string>
   </property>
  </widget>
  <widget class="QLabel" name="height_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Height:</string>
   </property>
  </widget>
  <widget class="QLabel" name="born_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Born in:</string>
   </property>
  </widget>
  <widget class="QLabel" name="sentence_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Sentence: </string>
   </property>
  </widget>
  <widget class="QLabel" name="dateOfConviction_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>510</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Date of Conviction:</string>
   </property>
  </widget>
  <widget class="QLabel" name="dateToBeLiberated_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>540</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Date to be liberated: </string>
   </property>
  </widget>
  <widget class="QDateEdit" name="Birthdate_dateEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>80</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::UpDownArrows</enum>
   </property>
   <property name="displayFormat">
    <string>yyyy - MM - dd</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDateEdit" name="DateOfConviction_dateEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>510</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::UpDownArrows</enum>
   </property>
   <property name="displayFormat">
    <string>yyyy - MM - dd</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDateEdit" name="DateToBeLiberated_dateEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>540</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::UpDownArrows</enum>
   </property>
   <property name="displayFormat">
    <string>yyyy - MM - dd</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="Sentence_textEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>260</y>
     <width>351</width>
     <height>231</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>Type sentence here 255 word max</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Name_lineEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="LastName_lineEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="Height_lineEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="ft_label">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>110</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ft.</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="EyeColor_lineEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>140</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="Born_lineEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>200</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="foto_label">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>20</y>
     <width>371</width>
     <height>311</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="addImage_pushButton">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>380</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Add image</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addPrisoner_pushButton">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>530</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Add Prisoner</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Close_pushButton">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>530</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QLabel" name="hair_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Hair Color:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Hair_lineEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>170</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Close_pushButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>actionAdd_prisoner</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>744</x>
     <y>547</y>
    </hint>
    <hint type="destinationlabel">
     <x>719</x>
     <y>510</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addPrisoner_pushButton</sender>
   <signal>clicked()</signal>
   <receiver>actionAdd_prisoner</receiver>
   <slot>add_prisoner_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>628</x>
     <y>562</y>
    </hint>
    <hint type="destinationlabel">
     <x>493</x>
     <y>530</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addImage_pushButton</sender>
   <signal>clicked()</signal>
   <receiver>actionAdd_prisoner</receiver>
   <slot>add_image_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>670</x>
     <y>398</y>
    </hint>
    <hint type="destinationlabel">
     <x>515</x>
     <y>384</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>add_prisoner_clicked()</slot>
  <slot>add_image_clicked()</slot>
 </slots>
</ui>
